src/state/slices/portfolioSlice/portfolioSlice.test.ts

```diff
@@ -48,6 +50,37 @@ import {
   selectTotalFiatBalanceWithDelegations,
 } from './selectors'
 
+const mockChainAdapters = new Map([
+  [
+    KnownChainIds.BitcoinMainnet,
+    {
+      getFeeAssetId: () => btcAssetId,
+    },
+  ],
+  [
+    KnownChainIds.CosmosMainnet,
+    {
+      getFeeAssetId: () => cosmosAssetId,
+    },
+  ],
+  [
+    KnownChainIds.EthereumMainnet,
+    {
+      getFeeAssetId: () => ethAssetId,
+    },
+  ],
+  [
+    KnownChainIds.AvalancheMainnet,
+    {
+      getFeeAssetId: () => avalancheAssetId,
+    },
+  ],
+])
+
+jest.mock('context/PluginProvider/PluginProvider', () => ({
+  getChainAdapters: () => mockChainAdapters,
+}))
```

Comment:
We could put this, and the duplicate mock logic in `src/state/slices/portfolioSlice/utils.test.ts` in a test helper file to reduce duplication.

Reply:
Nice one, agreed it would be better to have those mocks in a single source of truth. Moved to `test/mocks/portfolio` in 5c53e4a99df80d93d06c6bfe45fa890a1c646c4a